src/bundle/Resources/public/scss/ui/modules/muti-file-upload/_upload.item.scss

```diff
@@ -1,6 +1,6 @@
 .c-upload-list-item {
     display: flex;
-    flex-wrap: wrap;
+    flex-wrap: nowrap;
```

Comment:
With this fix I have very small warning icon
<img width="330" alt="Screenshot 2025-03-17 at 17 22 01" src="https://github.com/user-attachments/assets/c7d1e07d-fa07-45fd-969a-fb4ad9cd7440" />